backend/src/index.js

```diff
@@ -2,53 +2,77 @@
 // dotenv: access local environmental constants / variables
 require('dotenv').config();
 
+
 const express = require('express');
 const path = require('path');
 const app = express();
 
 // import mongoose & models from our Schema
-const {mongoose,User,NGO,affinities} = require("./Schema");
+const { mongoose, User, NGO, affinities } = require("./Schema");
 
 // Middleware
 app.use(express.json())
 app.use(express.urlencoded({ extended: false }))
 app.use(express.static(path.join(__dirname, 'src')))
+app.use('/ngos', (req, res, next) => {
```

Comment:
Lets move this middleware to the middleware folder as well, give it a name and import it here. Also we should just add this to the POST handler instead of the whole app as it might be used on other endpoints that share this path and we might not want this.

Reply:
so do we have to include it into the router.post(....)??